src/config.cpp

```diff
@@ -5,9 +5,9 @@
 #include <cstring>
 
 Config::Config() {
-  /* --- Set DEFAULT parameters of the config file options --- */
+  // Set DEFAULT parameters of the config file options --- 
```

Comment:
I have been using /* --- ... --- */ as a 'headline', to  visually separate the code. I'd appreciate if that stays so I find my stuff.